aptos-move/move-examples/defi/sources/locked_coins.move

```diff
@@ -0,0 +1,145 @@
+module defi::locked_coins {
+    use aptos_framework::account;
+    use aptos_framework::coin::{Self, Coin};
+    use aptos_framework::event::{Self, EventHandle};
+    use aptos_framework::timestamp;
+    use aptos_std::table::{Self, Table};
+    use std::error;
+    use std::signer;
+
+    /// Represents a lock of coins until some specified unlock time. Afterward, the recipient can claim the coins.
+    struct Lock<phantom CoinType> has store {
+        coins: Coin<CoinType>,
+        unlock_time_secs: u64,
+    }
+
```

Comment:
I get that the locks table is `<recipient_address, locked_coins of CoinType for this recipient>` but it'd be nice to have a comment making it clearer : )

Reply:
Updated